tensorboard/plugins/text_v2/text_demo_v2.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from absl import app
+from absl import logging
+import tensorflow as tf
+from tensorboard.plugins.text.summary_v2 import text
+
+tf.compat.v1.enable_v2_behavior()
+
+# Directory into which to write tensorboard data.
+LOGDIR = "/tmp/text_demo"
+
+# Number of steps for which to write data.
+STEPS = 16
+
+
+def simple_example(step):
+    # Text summaries log arbitrary text. This can be encoded with ASCII or
+    # UTF-8. Here's a simple example, wherein we greet the user on each
+    # step:
+    step_string = tf.as_string(step)
+    greeting = tf.strings.join(["Hello from step ", step_string, "!"])
+    text("greeting", greeting, step)
+
+
+def simple_example_with_pagination(step):
+    # Text summaries can be grouped by Tag (the first argument to the `text`
+    # summary's API).  Unique tags will have their own dropdown in the frontend,
+    # unless those tags are sub-tags of another tag.  To create a subtag name,
+    # one would input MainTagName/SubTagName as the input argument to the `text`
+    # summary API.  Subtags will be displayed within the main tag's dropdown,
+    # and a pager will appear if there are more than 12 subtags.  This test
+    # displays that UI feature by creating 36 subtags within the main tag:
+    # "simple_example_with_pagination"
+    for i in range(36):
+        step_string = tf.as_string(step)
+        greeting = tf.strings.join(["Hello from step ", step_string, "!"])
+        text("Card" + str(i + 1) + "/greeting", greeting, step)
```

Comment:
Use the Python string format for better readability. Similar to previous comment, basically.

Reply:
done